http/example/main.go

```diff
@@ -52,7 +52,7 @@ func itsHammerTime() int64 { return time.Now().UnixNano() }
 // Handler shares the initialized Responder across all example responses.
 type Handler struct {
 	*Responder
-	Ring ctx.KeyRingable
+	Ring keyring.Keyringable
```

Comment:
As far as I know, this is a naming convention we've run with (keyring/keyable). 1. I like it! 2. Are there any corollaries in django/rails land so that this language is a little more shared across web framworks.